plugins/datastreamer_plugin/CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ set (ALL_LIBRARY_SOURCES
         # UI files
         "ui/topic_selection_dialog/dialogselecttopics.cpp"
         "ui/topic_selection_dialog/dialogselecttopics.h"
+        "ui/topic_selection_dialog/Configuration.cpp"
```

Comment:
Only referring to header *.h files: are they really necessary here? Would cmake work just fine without them?

Reply:
I do not think they are necessary. I copy pasted from `plotjuggler-ros-plugins` and it does it this way, should I try to change it in this PR or we open a new one?